polaris-tokens/src/themes/base/color.ts

```diff
@@ -145,10 +145,16 @@ export type ColorBorderAlias =
   | 'border-tertiary'
   | 'border-warning'
   | 'border'
+  | 'border-gradient-experimental'
+  | 'border-gradient-hover-experimental'
+  | 'border-gradient-selected-experimental'
+  | 'border-gradient-active-experimental'
   /** Specialty and component border colors. */
   | 'input-border-active'
   | 'input-border-hover'
-  | 'input-border';
+  | 'input-border'
+  | 'tooltip-border-tail-down-experimental'
+  | 'tooltip-border-tail-up-experimental';
```

Comment:
These names don't quite follow the specialty token structure:
https://polaris.shopify.com/design/colors/color-tokens#specialty-tokens

e.g. `--p-color-<concept>-<optional-characteristic>-<default-token-structure>`

What do you think about:
- `--p-color-tooltip-tail-down-border-experimental`
- `--p-color-tooltip-tail-up-border-experimental`

cc @sarahill

Reply:
Took a look at the other specialty tokens and what you have here makes sense to me! updated it and its usages